functions/supplier/http-supplier/pom.xml

```diff
@@ -24,32 +24,6 @@
             <groupId>org.springframework.integration</groupId>
             <artifactId>spring-integration-webflux</artifactId>
         </dependency>
-
-        <dependency>
```

Comment:
Yep, good cleanup - these are all in spring-functions-parent (the validation ones are brought in via the validation starter as well).